### ISSUE TITLE
Video Speed menu is not accessible

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-media#297]
### Subject of the issue
The `_speed` player option is not accessible:

1. The speed selector button does not have a meaningful label for screenreaders. The button is announced by default as the "one point zero zero x button"
2. The speed selector button is not keyboard accessible. Mouse users can hover over the playback speed button to reveal more options, but when a keyboard users sets focus on the playback speed button and activates it with space or enter, nothing happens. Keyboard users are unable to change their playback speed.

In _mediaLibrariesOverrides.js_, we [override](https://github.com/adaptlearning/adapt-contrib-media/blob/a9f2bc7c252e772390b2533a1b7d228eb32e32c5/js/mediaLibrariesOverrides.js#L265) (or add?) aria labels for use with MediaElement.js. So, perhaps we could add a label for speed here.

The focus issue is similar to that of captions. See #232 

### Steps to reproduce
Add 'speed' to a Media component's player options. For example:

```
"_playerOptions": {
  "alwaysShowControls": false,
  "hideVideoControlsOnLoad": true,
  "showPosterWhenEnded": true,
  "features": [
    "playpause",
    "progress",
    "fullscreen",
    "volume",
    "speed"
  ]
},
```

Then, navigate to the option with a screenreader / keyboard navigation.

**Comment by swashbuck** [comment]
Created #298 to look into updating the MediaElement library.

**Comment by swashbuck** [comment]
Needs reviewing as part of #299

**Comment by kirsty-hames** [comment]
> Needs reviewing as part of [#299](https://github.com/adaptlearning/adapt-contrib-media/pull/299)

Similar to the [caption button issue](https://github.com/adaptlearning/adapt-contrib-media/issues/232#issuecomment-3601905875) the video speed list (menu for video speed selection) doesn't display when focused. The list only displays on hover.

This is due to the `.mejs__offscreen` class appended to `.mejs__speed-selector`.
